packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceDisconnectRequired.tsx

```diff
@@ -1,35 +1,31 @@
 import React from 'react';
-import { useSpring, config, animated } from 'react-spring';
 import styled from 'styled-components';
 
 import { Translation, TroubleshootingTips } from '@suite-components';
+import { motion } from 'framer-motion';
+import { enterEase } from '@suite-config/animation';
 
-const Wrapper = styled(animated.div)`
+const Wrapper = styled(motion.div)`
     display: flex;
 `;
 
-const DeviceDisconnectRequired = () => {
-    const fadeStyles = useSpring({
-        config: { ...config.default },
-        delay: 1000,
-        from: { opacity: 0 },
-        to: { opacity: 1 },
-    });
-
-    return (
-        <Wrapper style={fadeStyles}>
-            <TroubleshootingTips
-                label={<Translation id="TR_DISCONNECT_YOUR_DEVICE" />}
-                items={[
-                    {
-                        key: 'disconnect-your-device',
-                        heading: <Translation id="TR_DISCONNECT_YOUR_DEVICE" />,
-                        description: <Translation id="DISCONNECT_DEVICE_DESCRIPTION" />,
-                    },
-                ]}
-            />
-        </Wrapper>
-    );
-};
+const DeviceDisconnectRequired = () => (
+    <Wrapper
+        initial={{ opacity: 0 }}
+        animate={{ opacity: 1 }}
+        transition={{ delay: 0.6, duration: 0.5, ease: enterEase }}
```

Comment:
nit:
This wrapper with those animations is used at least in three simple components on the welcome page. Maybe it would be nice to create a Wrapper component for that or **merge all three components into a single one**?

Reply:
Yeah, I was considering another PR with animation changes all across the app, perhaps there are more things to unify. I will unify those here though. Also create a code cleanup commit :)

Reply:
64b7c6352ed771dfc7dc9a94a5c5c5f0554fca5e